tools/install.sh

```diff
@@ -1,10 +1,12 @@
 #!/bin/bash
 
 #
-# This will:
-#   1. Set up the LORIS DB schema
-#   2. Log the installation in the logs directory
-# This will only install the database components and LORIS config file.
+# This script will:
+#   1. verify composer is installed to manage dependencies
```

Comment:
This needs an edit given that `make` (run after this script) is the step that *installs* composer

Reply:
done : see my [commit 475ca8](https://github.com/aces/Loris/pull/5205/commits/475ca805073aba4296b34ac0698d3e16e4a5d3fa)